jdbc-company/src/dao/CompanyDaoImpl.java

```diff
@@ -0,0 +1,101 @@
+package dao;
+import com.bobocode.exception.DaoOperationException;
+import core.Company;
+
+import java.sql.*;
+import javax.sql.DataSource;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by hamster on 21.08.2017.
+ */
+public class CompanyDaoImpl implements CompanyDao {
+    private DataSource dataSource;
+
+    public CompanyDaoImpl(DataSource dataSource) {
+        this.dataSource = dataSource;
+    }
+
+    @Override
+    public void save(Company company) {
+        try {
+            Connection connection = dataSource.getConnection();
+            Statement statement = connection.createStatement();
+            PreparedStatement createCompanyString = prepareInsertStatement(connection, company);
+            statement.executeUpdate(createCompanyString.toString()); //вот тут непонятно что собственно делает в коде Тараса executeInsert
```

Comment:
I think it will be awesome to show as how we must did our home work. 
For example:
-do we need use existing classes? which of them? why?
-do we need create new module for our homework or use existing? why?
-how we can create pull request? To show it take only a  5 minutes.
I think it wil be very instructive for as. And we can take next level of knowledge for a simbolic price.